### DISCUSSION TITLE
How to display an empty map ?

### DISCUSSION THREAD

**Discussion by mff73** [mytourbook/mytourbook#1331] [Q&A] [answered]
I would like to share a screen shot of a tour on an empty map (for privacy reasons). I tried to create an empty map provider, but i also got tiles errors on the map (which is not nice 😊).
![image](https://github.com/mytourbook/mytourbook/assets/50132146/b6f21afe-c7ae-4381-9c7e-1a47215fcc6c)

Any other solution for this "privacy" sharing map question, please ?
thanks.

**Comment by wolfgang-ch** [comment, marked as answer]
You can dim the map but the track could still be used to find it in a tool which can search tracks.

![image](https://github.com/mytourbook/mytourbook/assets/1283445/cf12bd3f-dd73-427e-b448-6b1e05797f57)
